src/core/immersed_boundary/ibm_cuda.cu

```diff
@@ -129,20 +129,38 @@ void InitCUDA_IBM(const int numParticles)
     // Copy boundary velocities to the GPU
     // First put them into correct format
 #ifdef LB_BOUNDARIES_GPU
-    float* host_lb_boundary_velocity = new float[3*(n_lb_boundaries+1)];
+//    float* host_lb_boundary_velocity = new float[3*(n_lb_boundaries+1)];
```

Comment:
Please remove commented out code.  Don't worry about losing it, it will still be present in the history.